docs/logs.md

```diff
@@ -0,0 +1,253 @@
+# Logs
+
+We use server logs from the GOV.UK Design System website for basic analytics, we keep logs for 14 days and anonymise IP addresses.
+
+We store router logs ([GoRouter](https://docs.cloudfoundry.org/concepts/architecture/router.html)) and general application logs for 14 days and anonymise IP addresses.
+
+Our Content Delivery Network (CDN) logs are not currently available, this means many requests are not shown in the router logs.
+However, we do not currently cache HTML requests (pages) so router logs are still useful for pages visited.
+
+The GOV.UK Design System website's logs are sent to
+the [`GOV.UK Design System Production` Logstash stack (Logit.io)](https://logit.io/a/1c6b2316-16e2-4ca5-a3df-ff18631b0e74) using the `logit-ssl-drain` service.
+
+- the `logit-ssl-drain` service is bound to the website application in the [manifest services config option](../manifest.yml).
+- the hosted Logstash (Logit.io) is maintained by the Reliability Engineering team, we followed the [Logit setup guidance to configure our stack](https://reliability-engineering.cloudapps.digital/logging.html#get-started-with-logit).
+
+When the raw logs are sent to Logstash they are transformed using Logstash Filters so they can be queried in Kibana for visualisations and querying.
+
+## Logging into Logit.io
+
+Follow the Reliability engineering ['getting started with logit'](https://reliability-engineering.cloudapps.digital/logging.html#get-started-with-logit) instructions to login with your Google account.
+
+You can now access the [GOV.UK Design System Production stack](https://logit.io/a/1c6b2316-16e2-4ca5-a3df-ff18631b0e74).
+
+## How we transform logs with Logstash Filters
+
+The configuration is based on the [GOV.UK PaaS setup instructions](https://docs.cloud.service.gov.uk/monitoring_apps.html#configure-logstash-filters), see [git history for updates](https://github.com/alphagov/govuk-design-system/commits/master/docs/logs.md).
+
+**We use this file as version control for the Logit Filters for our stack. Please make sure to update the below code snippet in this file before changing Filters.**
```

Comment:
Think it's important there's code review before anyone applies new filters, probably pairing is the most practical, since if you break the filter you damage the data.